docs/building/ubuntu-instructions.md

```diff
@@ -179,39 +179,39 @@ Once you build the samples, you can use `spark-submit` to submit your .NET Core
      ```
 
      Here are some examples you can run:
-     - **[Microsoft.Spark.Examples.Sql.Basic](../../examples/Microsoft.Spark.CSharp.Examples/Sql/Basic.cs)**
+     - **[Microsoft.Spark.Examples.Batch.Basic](../../examples/Microsoft.Spark.CSharp.Examples/Batch/Basic.cs)**
```

Comment:
I can reintroduce the sql folder to maintain parity with the original Spark structure if we think that would be best. To me, I removed it since it was confusing that everything was in the sql folder anyways, so why introduce another unnecessary layer?

Reply:
@rapoth, @imback82   thoughts?

Reply:
We can do `Sql/Batch` and `Sql/Streaming` if it was confusing. (just `Streaming` is not enough since what we have is `StructuredStreaming` => `Sql.Streaming`.) `Sql` here implies the examples will be with `DataFrame`, so I don't think it's unnecessary.

Reply:
Addressed in new PRs.